frontend/src/metabase/query_builder/components/DataSelector/saved-question-picker/SavedQuestionPicker.unit.spec.jsx

```diff
@@ -104,10 +106,14 @@ describe("SavedQuestionPicker", () => {
   });
 
   it("sorts saved questions case-insensitive (metabase#23693)", async () => {
-    await setup();
+    const { fetchCollectionItemsMock } = await setup();
+
+    expect(fetchCollectionItemsMock.lastCall()[0]).toMatch(
+      "sort_column=name&sort_direction=asc",
```

Comment:
I wouldn't add checks for the parameters here, it should be an implementation detail. What we could do instead is to make `setupSearchEndpoints` take these parameters into account, so here we would only assert for the correct text content (done below)